### DISCUSSION TITLE
[Treeview] Is there any way to customize icon?

### DISCUSSION THREAD

**Discussion by cshimegi** [vuetifyjs/vuetify#14691] [Q&A] [answered]
Hi, everyone!

I try to customize the icon shown on the left side, but it seems there is no any elegant way to do that.
Does anyone know how to do this?

<img width="331" alt="スクリーンショット 2022-02-04 17 13 43" src="https://user-images.githubusercontent.com/67813918/152494553-3d7a1e62-b683-490c-8f06-9399273c912a.png">

**Comment by jacekkarczmarczyk** [comment, marked as answer]
https://vuetifyjs.com/en/api/v-treeview/#props-expand-icon

**Reply by cshimegi** [reply]
@jacekkarczmarczyk 
Thank you so much.
And sorry that I miss it.
